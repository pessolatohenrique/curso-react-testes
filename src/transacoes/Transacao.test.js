import React from "react";
import { render } from "@testing-library/react";
import Transacao from "./Transacao";

describe("Transacao component", () => {
  it("should render with snapshot", () => {
    const { container } = render(
      <Transacao data="01/01/2021" tipo="saque" valor="500" />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
