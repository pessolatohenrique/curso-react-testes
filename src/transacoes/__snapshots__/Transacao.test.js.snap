// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transacao component should render with snapshot 1`] = `
<div
  class="transacao-container"
>
  <p>
    01/01/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    500
  </p>
</div>
`;
